scripts/list_tiles.py

```diff
@@ -158,7 +158,7 @@ def check_args(argv):
 
     while (len(bounding_boxes) != 0):
       b_box = bounding_boxes.pop(0)
-      for level, tiles in tile_hierarchy.levels.items():
+      for level, tiles in list(tile_hierarchy.levels.items()):
```

Comment:
all of the changes inserting a `list` can be reverted IMO

Reply:
I guess it does that because in 2 it was lists returned by those functions, in 3 it's iterators/generators

Reply:
Yeah looks like `2to3` matched the types and converted it into a `list`. I'll remove the list conversions.